svf/lib/MSSA/MemSSA.cpp

```diff
@@ -599,7 +599,7 @@ void MemSSA::dumpMSSA(OutStream& Out)
             }
         }
 
-        for (SVFFunction::const_iterator bit = fun->begin(), ebit = fun->end();
+        for (SVFFunction::svfbb_iterator bit = fun->begin(), ebit = fun->end();
```

Comment:
why changed this to svfbb_iterator? Is this a const_iterator or you have a non-const one?

Reply:
Because need to add SVFBasicBlock to SVFFunction when handle external 'CallStmt', the  
"std::vector<const SVFBasicBlock*> allBBs;" has been changed to 
"std::vector<SVFBasicBlock*> allBBs;". Therefore, the iterator also needs to be modified.